src/chrome/content/rules/clodo.ru.xml

```diff
@@ -0,0 +1,267 @@
+184691-20006.clodo.ru ¹
+184691-20007.clodo.ru ⁷
+184691-20008.clodo.ru ²
+1868-7.clodo.ru ⁷
+1888-3.clodo.ru ³
+1888-4.clodo.ru ⁷
+62.76.188.2.clodo.ru ¹
+21-757.clodo.ru ³
+21-758.clodo.ru ³
+21-759.clodo.ru ³
+21-760.clodo.ru ³
+21-761.clodo.ru ³
+21-762.clodo.ru ³
+21-763.clodo.ru ³
+21-764.clodo.ru ³
+21-765.clodo.ru ³
+21-767.clodo.ru ³
+21-768.clodo.ru ³
+21-769.clodo.ru ³
+21-770.clodo.ru ³
+21-771.clodo.ru ³
+21-772.clodo.ru ³
+21-790.clodo.ru ⁷
+210066-6.clodo.ru ²
+211951-5.clodo.ru ²
+212153-1.clodo.ru ¹
+215894-10002.clodo.ru ⁷
+217734-1.clodo.ru ⁵
+217768-10058.clodo.ru ⁷
+217768-175.clodo.ru ⁷
+217768-20002.clodo.ru ⁷
+217768-336.clodo.ru ⁷
+217768-36.clodo.ru ⁷
+217768-397.clodo.ru ⁷
+217768-72.clodo.ru ⁷
+217768-8.clodo.ru ⁷
+218086-10001.clodo.ru ⁷
+218294-20001.clodo.ru ⁷
+218460-10003.clodo.ru ⁴
+218708-20002.clodo.ru ⁷
+218804-10001.clodo.ru ⁴
+218998-10001.clodo.ru ²
+219406-10002.clodo.ru ⁴
+219612-2.clodo.ru ²
+219968-1.clodo.ru ⁴
+220220-1.clodo.ru ⁷
+220482-20001.clodo.ru ²
+220740-20001.clodo.ru ⁷
+220966-10001.clodo.ru ⁴
+221252-20001.clodo.ru ⁷
+221392-20001.clodo.ru ⁷
+221604-20003.clodo.ru ⁷
+221618-10001.clodo.ru ⁷
+221656-10002.clodo.ru ²
+222168-1.clodo.ru ⁷
+222336-10002.clodo.ru ⁴
+222372-10001.clodo.ru ⁷
+222402-10001.clodo.ru ⁷
+222446-1.clodo.ru ²
+222880-10001.clodo.ru ⁷
+223200-10001.clodo.ru ⁴
+85.143.166.224.clodo.ru ¹
+37-139-47-162.clodo.ru ⁷
+37-139-47-182.clodo.ru ⁷
+37-139-47-83.clodo.ru ⁷
+37581-4.clodo.ru ⁷
+43951-3.clodo.ru ⁷
+5591-10763.clodo.ru ²
+5591-20259.clodo.ru ⁷
+62.76.41.58.clodo.ru ¹
+62.76.40.59.clodo.ru ¹
+6-568.clodo.ru ⁷
+6-585.clodo.ru ²
+62-76-178-11.clodo.ru ⁷
+62-76-179-48.clodo.ru ⁷
+62-76-180-142.clodo.ru ⁷
+62-76-180-195.clodo.ru ⁷
+62-76-180-75.clodo.ru ⁷
+62-76-181-213.clodo.ru ⁷
+62-76-182-4.clodo.ru ⁷
+62-76-182-7.clodo.ru ⁷
+62-76-182-8.clodo.ru ⁷
+62-76-184-113.clodo.ru ⁷
+62-76-184-57.clodo.ru ⁷
+62-76-185-199.clodo.ru ⁷
+62-76-186-235.clodo.ru ⁷
+62-76-186-62.clodo.ru ⁷
+62-76-187-45.clodo.ru ⁷
+62-76-188-167.clodo.ru ⁷
+62-76-188-211.clodo.ru ⁷
+62-76-188-231.clodo.ru ⁷
+62-76-188-240.clodo.ru ⁷
+62-76-188-242.clodo.ru ⁷
+62-76-188-41.clodo.ru ⁷
+62-76-188-66.clodo.ru ⁷
+62-76-189-129.clodo.ru ⁷
+62-76-189-145.clodo.ru ⁷
+62-76-189-181.clodo.ru ⁷
+62-76-189-247.clodo.ru ⁷
+62-76-189-31.clodo.ru ⁷
+62-76-189-47.clodo.ru ⁷
+62-76-189-61.clodo.ru ⁷
+62-76-190-13.clodo.ru ⁷
+62-76-190-181.clodo.ru ⁷
+62-76-191-100.clodo.ru ⁷
+62-76-191-81.clodo.ru ⁷
+62-76-40-177.clodo.ru ⁷
+62-76-41-15.clodo.ru ⁷
+62-76-41-219.clodo.ru ⁷
+62-76-42-106.clodo.ru ⁷
+62-76-42-67.clodo.ru ⁷
+62-76-43-133.clodo.ru ⁷
+62-76-43-139.clodo.ru ⁷
+62-76-43-140.clodo.ru ⁷
+62-76-43-165.clodo.ru ⁷
+62-76-43-53.clodo.ru ⁷
+62-76-43-78.clodo.ru ⁷
+62-76-45-223.clodo.ru ⁷
+62-76-46-153.clodo.ru ⁷
+62-76-47-249.clodo.ru ⁷
+69421-1.clodo.ru ⁴
+72151-2.clodo.ru ⁴
+85-143-166-11.clodo.ru ⁷
+85-143-166-120.clodo.ru ⁷
+85-143-166-154.clodo.ru ⁷
+85-143-166-6.clodo.ru ⁷
+85-143-166-7.clodo.ru ⁷
```

Comment:
Is there any orderliness between all those numeric subdomains that we can avoid that huge list?